src/main/java/net/tiagofar78/prisonescape/managers/MessageLanguageManager.java

```diff
@@ -219,6 +223,16 @@ public List<String> getGameStartingAnnouncementMessage(int remainingTime, int pl
 		
 		return message;
 	}
+
+	public String getPrisionerArrested(String playerName) {
+		String message = new String(_prisionerArrested);
+		message.replace("{PLAYER}", playerName);
```

Comment:
Replace method for strings doesn't modify the current string, it returns a new one modified.